CorsixTH/Lua/game_ui.lua

```diff
@@ -91,6 +91,25 @@ function GameUI:GameUI(app, local_hospital, map_editor)
 
   self.speed_up_key_pressed = false
 
+  -- The currently specified intensity value for earthquakes. To abstract
+  -- the effect from the implementation this value is a number between 0
+  -- and 1.
+  self.shake_screen_intensity = 0
+
+  self.announcer = Announcer(app)
+end
+
+function GameUI:setupGlobalKeyHandlers()
+  UI.setupGlobalKeyHandlers(self)
+
+  -- Set the scrolling keys.
+  self.scroll_keys = {
+     [tostring(self.app.hotkeys["ingame_scroll_up"])] = {x = 0, y = -10},
+     [tostring(self.app.hotkeys["ingame_scroll_down"])] = {x = 0, y = 10},
+     [tostring(self.app.hotkeys["ingame_scroll_left"])] = {x = -10, y = 0},
+     [tostring(self.app.hotkeys["ingame_scroll_right"])]	= {x = 10, y = 0},
```

Comment:
Bit much whitespace before the `=`

Reply:
That block was preexisting, just moved. I'm ok either way.